jenkins_build.sh

```diff
@@ -77,6 +77,7 @@ source $(zendev bootstrap)
 
 echo Creating a zendev environment...
 cd $WORKSPACE
+zendev drop ${ZENDEV_ENV}
```

Comment:
If `${ZENDEV_ENV}` does not exist, will zendev throw an error and cause the entire script to stop?